apple/REAModule.mm

```diff
@@ -158,47 +159,35 @@ - (void)handleJavaScriptDidLoadNotification:(NSNotification *)notification
 
 /*
  * Taken from RCTNativeAnimatedTurboModule:
- * In bridgeless mode, `setBridge` is never called during initialization. Instead this selector is invoked via
- * BridgelessTurboModuleSetup.
+ * This selector is invoked via BridgelessTurboModuleSetup.
  */
 - (void)setSurfacePresenter:(id<RCTSurfacePresenterStub>)surfacePresenter
 {
   _surfacePresenter = surfacePresenter;
+  _isBridgeless = true;
 }
 
 - (void)setBridge:(RCTBridge *)bridge
```

Comment:
It's a bit confusing that we have two separate places where we define `setBridge:` method (line 170 and line 207). I know it was like that prior to this PR. It's definitely not a blocker but we should do something about it in a separate PR.